src/services/chris_integration.tsx

```diff
@@ -32,7 +33,8 @@ interface PACSFile {
 enum PluginPollStatus {
```

Comment:
Could you do me a favor and add another state called "REGISTERING_FILES"?

I just check DB migrations and as of Feb 9, 2021, the possible states are as follows:
`field=models.CharField(choices=[('created', 'Default initial'), ('waiting', 'Waiting to be scheduled'), ('scheduled', 'Scheduled on worker'), ('started', 'Started on compute env'), ('registeringFiles', 'Registering output files'), ('finishedSuccessfully', 'Finished successfully'), ('finishedWithError', 'Finished with error'), ('cancelled', 'Cancelled')], default='created', max_length=30)`

Reply:
Added